ci-operator/step-registry/mirror-images/by-oc-mirror/mirror-images-by-oc-mirror-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: mirror-images-by-oc-mirror
```

Comment:
Let us create a new install workflow to use this new chain in `ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/by-oc-mirror/sno/`, you can reference `aws/ipi/disconnected/private/sno/` as example.